pages/settings/account/index.tsx

```diff
@@ -268,10 +270,14 @@ const AccountSettings = () => {
     }
   })
 
-  const loading = userInfoLoading || status === 'loading'
+  const loading = userInfoLoading || status === SessionStatus.Loading
```

Comment:
Hmmm we shouldnt be making a graphql request for useUserInfoQuery when the user is unauthenticated, that makes no sense